metaflow/task.py

```diff
@@ -366,14 +366,6 @@ def run_step(
         output.init_task()
 
         if input_paths:
```

Comment:
Ditto in terms of stripping out just control tasks. I am curious to know if we can support both modes.

Reply:
Yeah, so it was just arbitrary that conda was not supported for control tasks. Now it is.